src/core/infer_trace.h

```diff
@@ -46,13 +46,24 @@ class InferenceTrace {
   InferenceTrace(
       const TRITONSERVER_InferenceTraceLevel level, const uint64_t parent_id,
       TRITONSERVER_InferenceTraceActivityFn_t activity_fn,
-      TRITONSERVER_InferenceTraceReleaseFn_t release_fn, void* userp)
+      TRITONSERVER_InferenceTraceReleaseFn_t release_fn, void* userp,
+      TRITONSERVER_InferenceTraceTensorActivityFn_t tensor_activity_fn =
```

Comment:
Move tensor_activity_fn argument to after activity_fn. You will not be able to use a default augument value (nullptr) so you will need to change the callers to explicitly pass nullptr.